src/commcare_cloud/ansible/roles/redis/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- include: setup_cluster.yml
+- include_tasks: setup_cluster.yml
   when: is_redis_cluster|bool and inventory_hostname in groups['redis_cluster_master']
 
```

Comment:
the task is dynamic because its inclusion depends on the values of is_redis_cluster and the presence of the Ansible hostname in the redis_cluster_master group.